bigcommerce/customer_login_token.py

```diff
@@ -0,0 +1,55 @@
+import os
+import time
+import uuid
+import jwt
+
+# Get the client_secret needed to sign tokens from the environment
+# Intended to play nice with the Python Hello World sample app
+# https://github.com/bigcommerce/hello-world-app-python-flask
+client_secret = os.getenv('APP_CLIENT_SECRET', None)
+
+
+class CustomerLoginTokens:
```

Comment:
I think the current recommendation is to always do `class Foo(object):` explicitly